src/sctools/test/test_metrics.py

```diff
@@ -9,6 +9,14 @@
 from sctools.metrics.gatherer import GatherGeneMetrics, GatherCellMetrics
 from sctools.platform import TenXV2
 
+# ############################### Testing Ground Truth Values ################################### #
```

Comment:
Minor style suggestion: you could use a `"""triple-quoted string here for this comment block"""`